Dockerfile

```diff
@@ -27,9 +27,7 @@ COPY . .
 ENV PATH="$PATH:/home/ubuntu"
 
 # Service-specific configurations
-EXPOSE 8891
```

Comment:
I think we still need this for ingress?

Reply:
On kube I am running everything on 8892 now, 8891 is only for local dev ingress.

Reply:
Aaah right, I forgot ingress is in its own service now, so a separate port makes no sense at all!